### PR TITLE
Update androidx support section with important points.

### PR DESCRIPTION
**Pull request by helpshift-tanuj**

### CODE REVIEW

File: docs/sdkx-unity/troubleshooting-android.mdx [hunk 1]
```diff
@@ -29,40 +29,45 @@ Helpshift Unity SDK X uses legacy android support libraries.
 If you want to use AndroidX libraries in your application please follow these steps to make Helpshift SDK compatible with AndroidX:
 
 1. We recommend to use the standalone jetifier tool mentioned [here](https://developer.android.com/studio/command-line/jetifier).
-1. To install Jetifier, [download the zip file](https://dl.google.com/dl/android/studio/jetifier-zips/1.0.0-beta07/jetifier-standalone.zip) and extract it. Your development environment must have Java version 1.8 installed.
-1. Use `standalone jetifier` tool to manually jetify the Helpshift library files:
+1. To install Jetifier, [download the zip file](https://dl.google.com/dl/android/studio/jetifier-zips/1.0.0-beta07/jetifier-standalone.zip) and extract it. Your development environment must have **Java version 1.8** installed.
```
Comment (hs-saunved) [inline docs/sdkx-unity/troubleshooting-android.mdx:32]: Please update the bullet points to be 1, 2, 3, 4...

I am aware that they render correctly in the final output, but it's a good idea to patch them up here.

File: docs/sdkx-unity/troubleshooting-android.mdx [hunk 2]
```diff
@@ -29,40 +29,45 @@ Helpshift Unity SDK X uses legacy android support libraries.
...
-compile 'com.google.android.material:material:1.0.0'
-...
+  ...
+  implementation 'com.google.android.material:material:1.0.0'
+  ...
 }
 ```
 
-After all these changes, run a clean build of your app and verify all the UI screens work as expected.
+After all these changes, **ensure there are no legacy android support libraries present** in your project. Build your app and verify if all the UI screens are working as expected.
```
Comment (hs-saunved) [inline docs/sdkx-unity/troubleshooting-android.mdx:70]: "android" => "Android" (capitalize)

File: docs/sdkx-unity/troubleshooting-android.mdx [hunk 3]
```diff
@@ -44,7 +44,7 @@ Example:
 
 If you are using **`Unity Jar Resolver`** plugin to resolve support library requirements:
 
-- Update the `<project>/Assets/Helpshift/Editor/HelpshiftDependencies.xml` to only use the androidx library and remove legacy android support library:
+- Update the `<project>/Assets/Helpshift/Editor/HelpshiftDependencies.xml` to only use the androidx library and remove legacy Android support library:
```
Comment (hs-saunved) [inline docs/sdkx-unity/troubleshooting-android.mdx:47]:
```suggestion
- Update the `<project>/Assets/Helpshift/Editor/HelpshiftDependencies.xml` to only use the AndroidX library and remove legacy Android support library:
```

File: docs/sdkx-unity/troubleshooting-android.mdx [hunk 4]
```diff
@@ -57,7 +57,7 @@ If you are using **`Unity Jar Resolver`** plugin to resolve support library requ
 
 If you are using **`Gradle`** to resolve support library requirements:
 
-- Update dependencies section of the `<project>/Assets/Plugins/Android/mainTemplate.gradle` or `build.gradle` file to use the androidx material library as:
+- Update dependencies section of the `<project>/Assets/Plugins/Android/mainTemplate.gradle` or `build.gradle` file to use the Androidx material library as:
```
Comment (hs-saunved) [inline docs/sdkx-unity/troubleshooting-android.mdx:60]:
```suggestion
- Update dependencies section of the `<project>/Assets/Plugins/Android/mainTemplate.gradle` or `build.gradle` file to use the AndroidX material library as:
```

**Review by hs-saunved — CHANGES_REQUESTED** [review]
Some nitpicks

**Review by hs-saunved — APPROVED** [review]

**Review by jaydeep-helpshift — APPROVED** [review]

**Pull request merged**